src/renderer/actions/connections.js

```diff
@@ -61,12 +61,12 @@ export function connect (id, databaseName, reconnecting = false, sshPassphrase)
       dispatch({ type: CONNECTION_REQUEST, server, database, reconnecting, isServerConnection: !databaseName });
 
       if (!serverSession) {
-        if (server.ssh.privateKeyWithPassphrase && typeof sshPassphrase === 'undefined') {
+        if (server.ssh && server.ssh.privateKeyWithPassphrase && typeof sshPassphrase === 'undefined') {
           dispatch({ type: CONNECTION_REQUIRE_SSH_PASSWORD });
           return;
         }
 
-        if (server.ssh.privateKeyWithPassphrase) {
+        if (server.ssh && server.ssh.privateKeyWithPassphrase) {
```

Comment:
Seems this fixing is not related to this issue. Do you mind moving them to a separated PR?